charts/graph-node/dashboards/indexing-status-overview.json

```diff
@@ -712,7 +712,7 @@
+          "rawSql": "-- grafana ignore\r\nselect\r\n  m.repository as repo,\r\n  g.name || (case\r\n        when g.pending_version = v.id then ' (P)'\r\n        when g.current_version = v.id then ' (C)'\r\n        else ' (U)' end) as subgraph_name,\r\n  d.deployment,\r\n  s.id as schema,\r\n  a.node_id,\r\n  n.name as network,\r\n  (n.head_block_number - d.latest_ethereum_block_number) as blocks_behind_network,\r\n  (case when n.name = 'mainnet' then ((n.head_block_number - d.latest_ethereum_block_number)/4)::text\r\n        when n.name = 'rinkeby' then ((n.head_block_number - d.latest_ethereum_block_number)/4)::text\r\n        when n.name = 'kovan' then ((n.head_block_number - d.latest_ethereum_block_number)/15)::text\r\n        when n.name = 'poa-core' then ((n.head_block_number - d.latest_ethereum_block_number)/12)::text\r\n        when n.name = 'ropsten' then ((n.head_block_number - d.latest_ethereum_block_number)/2)::text\r\n        else 'ø' end) as lag,\r\n        d.failed,\r\n        d.latest_ethereum_block_number,\r\n        a.paused_at\r\nfrom subgraphs.subgraph_deployment as d,\r\n     subgraphs.subgraph_deployment_assignment as a,\r\n     subgraphs.subgraph_version as v,\r\n     subgraphs.subgraph as g,\r\n     subgraphs.subgraph_manifest as m,\r\n     ethereum_networks  as n,\r\n     deployment_schemas as s\r\nwhere g.id = v.subgraph\r\n  and v.id in (g.pending_version, g.current_version)\r\n  and a.id = s.id\r\n  and s.id = d.id\r\n  and m.id = d.id\r\n  and v.deployment = d.deployment\r\n  /*and not d.failed*/\r\n  and n.name = s.network\r\n  and not a.node_id = 'removed'\r\norder by blocks_behind_network desc, subgraph_name;",
```

Comment:
removing `d.synced` as it has been removed in latest versions in favour of [`synced_at`](https://github.com/graphprotocol/graph-node/pull/5566/files#diff-ec890e6af3c4fe87135cd0099bb53db95954a0f4cb3a746a19e450dcbf4601c3L162)  which shows `1970-01-01` and [`synced_at_block_number`](https://github.com/graphprotocol/graph-node/pull/5610/files#diff-95524a308d634f9d04f085a829a9aeaae1c863dd69a0cd4e05af5c845dd6c66cR136) which currently shows as an empty value. Since we can roughly tell whether a subgraph is synced by how many blocks behind it is I think removing this field is a good option. @chriswessels lmk if you agree